picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +436,29 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+
+    /**
+     * Set a list of transformations to be applied to the image.  These transformations will
+     * overwrite any existing transformations.
```

Comment:
Is this the behavior we want? Or should it just append? I would have thought the latter (which probably would require it to be named `addTransformations`)